src/modules/likesApiConfig.js

```diff
@@ -0,0 +1,56 @@
+const LIKE_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Nbc4s2TFp5CN6zapCfwg/likes';
+let likeBtns;
+let likeBadges;
+
+const getLikes = async () => {
+  const response = await fetch(LIKE_URL);
+  const data = await response.json();
+  return data;
+};
+
+const displayLikes = () => {
+  likeBadges = document.querySelectorAll('.like-count');
+  getLikes().then((data) => {
+    data.forEach((like) => {
+      likeBadges.forEach((badge) => {
+        if (Number(badge.id) === Number(like.item_id)) {
+          badge.innerHTML = like.likes;
+        }
+      });
+    });
+  });
+};
+
+const postLikes = async (id) => {
+  const response = await fetch(LIKE_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+    }),
+  });
+  return response.json();
+};
+
+const submitLikes = () => {
+  likeBtns = document.querySelectorAll('.like-btn');
+  likeBtns.forEach((btn) => {
+    btn.addEventListener('click', async (event) => {
+      event.stopPropagation();
+      // eslint-disable-next-line prefer-destructuring
```

Comment:
- [ ] The same here, Please fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉